src/index.js

```diff
@@ -887,10 +887,21 @@ class RedContentScript extends ContentScript {
 
   async checkPersonnalInfosLinkVisibility() {
     this.log('info', '📍️ checkPersonnalInfosLinkVisibility starts')
-    const elementComputedStyles = window.getComputedStyle(
-      document.querySelector(`a[href="${PERSONAL_INFOS_URL}"]`)
+    const infoPersoElement = document.querySelector(
+      `a[href="${PERSONAL_INFOS_URL}"]`
+    )
+    let isVisible = false
+    if (infoPersoElement) {
+      this.log('info', 'Found link')
+      const elementComputedStyles = window.getComputedStyle(
+        document.querySelector(`a[href="${PERSONAL_INFOS_URL}"]`)
```

Comment:
You forgot to replace `document.querySelector(`a[href="${PERSONAL_INFOS_URL}"]`)` with `infoPersoElement` here

Reply:
Indeed 🤦‍♂️ 
 thanks !